viz.Rmd

```diff
@@ -85,82 +94,172 @@ making it easier for them to understand and remember your message quickly.
-records the atmospheric concentration of carbon dioxide (CO2, in parts per million) at the Mauna Loa research station in \index{Mauna Loa CO2} Hawaii from 1959 onwards [@maunadata].
+To get started, we read in and take a look at the data:
 
-**Question:** \index{question!visualization} Does the concentration of atmospheric CO2 change over time, and are there any interesting patterns to note?
 ```{r 03-data-co2, warning=FALSE, message=FALSE}
 # mauna loa carbon dioxide data
-co2_df <- read_csv("data/mauna_loa.csv") |>
-  filter(ppm > 0, date_decimal < 2000)
+co2_df <- read_csv("data/mauna_loa_data.csv")
 co2_df
 ```
 
-Since we are investigating a relationship between two variables (CO2 concentration and date), a scatter plot is a good place to start. Scatter plots 
-show the data as individual points with `x` (horizontal axis) and `y` (vertical axis) coordinates. Here, we will use the decimal
- date as the `x` coordinate 
-and CO2 concentration as the `y` coordinate. When using the `ggplot2` package, we create the plot object with the `ggplot` function; there are 
-a few basic aspects of a plot that we need to specify:
+We see we have two columns available to us; `date_measured` and `ppm`. 
+The `date_measured` column holds the date the measurement was taken, 
+and is of type `date`.
+The `ppm` columns holds the value of $CO_2$ in parts per million 
+that was measured on each date, and is type `double`.
+
+> **Note:** `read_csv` was able to parse the `date_measured` column into the
+> `date` vector type because it was entered 
+> in the international standard date format, 
+> called ISO 8601, which lists dates as `year-month-day`.
+> `date` vectors are `double` vectors with special properties that allow 
+> them to handle dates correctly.
+> For example, `date` type vectors allow functions like `ggplot` 
+> to treat them as numeric dates and not a character vectors, 
+> even though they contain non-numeric characters 
+> (e.g., `-` in the `date_measured` column in the `co2_df` data frame).
+> This means R it will not accidentally plot the dates in the wrong order 
+> (i.e., not alphanumerically as would happen if it was a character vector). 
+> An in depth study of dates and times is beyond the scope of the book, 
+> but we invite readers who are interested to learn more 
+> to see the Dates and times chapter of R for Data Science [@wickham2016r]
+> (linked to in our Additional Resources section at the end of this chapter).
+
+Since we are investigating a relationship between two variables 
+(CO$_{\text{2}}$ concentration and date), 
+a scatter plot is a good place to start. 
+Scatter plots show the data as individual points with `x` (horizontal axis) 
+and `y` (vertical axis) coordinates.
+An example of a scatter plot is shown in Figure \@ref(fig:scatter-plot-sketch).
+
+```{r scatter-plot-sketch, echo = FALSE, fig.width = 3.25, fig.height = 3, fig.align = 'center', fig.cap = "Example of a scatter plot."}
```

Comment:
I really like these plots you created and I think these are so great for explaining the basic plots types! I think we should move all of these plots up to section 4.3: Choosing the visualization. Right after we introduce the list of plots we can give a visual representation of the plots with these figures you've created. It makes the list in 4.3 less abstract and also allows people to compare the visualizations. It also makes these sections less distracting so we can focus only on the question  we want to answer about the data + iterating as opposed to learning what a scatterplot is *and* trying to answer the question